test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/SessionTests.cs

```diff
@@ -194,14 +194,17 @@ public void CreateRowNotInitialized()
                 new EditColumnMetadata                   // No default
                 {
                     DefaultValue = null,
-                    EscapedName = cols[0].ColumnName,
+                    EscapedName = cols[0].ColumnName
                 },
                 new EditColumnMetadata                   // Has default
                 {
                     DefaultValue = "default",
-                    EscapedName = cols[0].ColumnName,
+                    EscapedName = cols[1].ColumnName
                 },
-                new EditColumnMetadata()
+                new EditColumnMetadata
```

Comment:
Why this change? Will real columns always have an `EscapedName`?

Reply:
I haven't seen any cases where it's empty, and the rest of the code just seems to assume it's always there. This test case also fails in the new filtering code path if there's no column name.

Reply:
Yep, IIRC, a column name is always provided by the server. Worst case it's an empty string, but it's never null, IIRC.